addons/binding/org.openhab.binding.netatmo/ESH-INF/thing/welcomeCamera.xml

```diff
@@ -0,0 +1,113 @@
+            <channel id="welcomeCameraVpnUrl" typeId="vpn_url"></channel>
+            <channel id="welcomeCameraIsLocal" typeId="is_locale"></channel>
+                                    
+        </channels>
+              
+        <properties>
+             <property name="actualApp">app_welcomecamera</property>
+        </properties>         
+        
+        <config-description>
+            <parameter name="welcomeHomeId" type="text" required="true" readOnly="true">
+            	<label>Home ID</label>
+            	<description>Id of the home</description>
+            </parameter>
+
+            <parameter name="welcomeCameraId" type="text" required="true" readOnly="true">
+            	<label>Person ID</label>
+            	<description>Id of the camera</description>
+            </parameter>
+
+        </config-description>      
+	                          
+    </thing-type>
+
+                  
+    <channel-type id="id">
+        <item-type>String</item-type>
+        <label>Id</label>
+        <description>ID of the camera</description>
+        <state readOnly="true">
+        </state>
+    </channel-type>
+    <channel-type id="type">
+        <item-type>String</item-type>
+        <label>Type</label>
+        <description>Type of the camera</description>
+        <state readOnly="true"></state>
+    </channel-type>
+    <channel-type id="status">
+        <item-type>String</item-type>
+        <label>State</label>
+        <description>State of the camera</description>
+        <state readOnly="true">
+            <options>
+                <option value="on">on</option>
+                <option value="off">off</option>
+            </options>
+        </state>       
+    </channel-type>
+    <channel-type id="sd_status">
+        <item-type>String</item-type>
+        <label>SD State</label>
+        <description>State of the SD card</description>
+        <state readOnly="true">
+            <options>
+                <option value="on">on</option>
+                <option value="off">off</option>
+            </options>
+        </state>   
+    </channel-type>
+    <channel-type id="alim_status">
+        <item-type>String</item-type>
+        <label>Alim State</label>
+        <description>State of the power connector</description>
+        <state readOnly="true">
+            <options>
+                <option value="on">on</option>
+                <option value="off">off</option>
+            </options>
+        </state>   
+    </channel-type>
+    <channel-type id="name">
+        <item-type>String</item-type>
+        <label>Camera name</label>
+        <description>Name of the camera</description>
+        <state readOnly="true"></state>
+    </channel-type>
+    <channel-type id="vpn_url">
+        <item-type>String</item-type>
+        <label>VPN Url</label>
+        <description>Only for scope access_camera. Address of the camera</description>
```

Comment:
> maybe i also could handle the operations described on the netatmo site (Local vs remote)

I think that would be worthwhile!  The description of the [Welcome Ping Method](https://dev.netatmo.com/dev/resources/technical/reference/welcome/ping) seems very straightforward, so if you had a `snapshot_url` channel, it would be derived from `local_url` if openHAB and the camera are on the same local network.  Otherwise, it would be derived from the `vpn_url`.  The openHAB user would want the local path if available, remote path if not, but either way, the binding would do the work to produce the usable `snapshot_url` channel.